docs/git-guide.md

```diff
@@ -1497,3 +1498,4 @@ Deleting local branch review-original-5156 (was 5a1e982)
 [understanding-git]: http://web.mit.edu/nelhage/Public/git-slides-2009.pdf
 [edx-howto-rebase-pr]: https://github.com/edx/edx-platform/wiki/How-to-Rebase-a-Pull-Request
 [tig]: http://jonas.nitro.dk/tig/
+[wip-prs]: git-guide.html#work-in-progress-pull-requests
```

Comment:
it does work without the `git-guide.html`, i had it there because a couple of other links on this page have that structure. since it's redundant, should i PR an update that cleans this up for all files or is that not necessary?